cmd/juju/status/status.go

```diff
@@ -387,15 +387,15 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 		// A change was made in cmd/v3.0.2 output.go that broke the consistency in output for the
 		// default formatter by removing the newline delimiter. Hence we prefix '\n' in the text below.
 		// https://github.com/juju/cmd/commit/be22fa661a798055c801f1511aee226db249ef95
-		ctx.Infof("\nModel %q is empty.", modelName)
+		_, _ = fmt.Fprintf(ctx.Stdout, "\nModel %q is empty.\n", modelName)
```

Comment:
since we are ignoring the number of written bytes and a possible error, I think you can simply use:
```
fmt.Fprintf(ctx.Stdout, "\nModel %q is empty.\n", modelName)
```
(and the same on line 398)

Reply:
We generally like to include `_`s when we're ignoring return values, just to make it explicit that we're ignoring them